Sources/SwiftKuery/QueryBuilder.swift

```diff
@@ -57,11 +90,11 @@ public class QueryBuilder {
         case double
         /// The database type that corresponds to char.
         case char
-        
         /// Last case, add new values before it.
```

Comment:
it this supposed to be like this?

Reply:
You didn't answer Enrique's question here - it this supposed to be like this?